lib/features/property_detail/widgets/property_features_view.dart

```diff
@@ -70,14 +70,15 @@ class _PropertyFeaturesViewState extends State<PropertyFeaturesView>
                                 const SizedBox(
                                   width: HsDimens.spacing12,
                                 ),
-                                Text(
+                                Expanded(
+                                    child: Text(
                                   e.displayName,
                                   style: Theme.of(context)
                                       .textTheme
                                       .bodyMedium
                                       ?.copyWith(color: HSColor.neutral9),
                                   overflow: TextOverflow.ellipsis,
```

Comment:
Need max line for TextOverflow to take effect

Reply:
TextOverFlow already effected the context due to by default maxLines of Text widget is 1, I already discussed to PO about this and she also approved with maxLines = 1.

Reply:
Already added